### ISSUE TITLE
Library scan not picking up converted file formats

### ISSUE THREAD

**Issue by Shadow229** [Kareadita/Kavita#1778]
**Describe the bug**
Downloaded books in EPUB file format drop initially into calibre and get converted into AZW3 for use with kindle, the folder this happens in is watched by Kavita to scan for its library. However, only the original EPUB file format gets picked up by Kavita. If after the conversion (I'd have both epub and azw3 formats at this point) I delete the EPUB file, the book disappears entirely from Kavita and a library scan will not pick up the book at all. Equally, if I delete a book from Kavita's library page but the file still exists (even an epub that worked fine on initial scan) in the folder structure, a re scan does not pick it back up again (I think this might be the same behaviour without the need to convert to test?)

**To Reproduce**
1. Import a new .EPUB book in a normal folder structure (library/author/book.epub)
2. Note that book gets loaded into Kavita with folder watching or with a scan.
3. Convert the epub to a azw3 file (using calibre for this leaves you with both .epub and .azw3 files in the library/author/ folder)
4. Do a scan - Check no other version has pulled through into Kavita (my file type on the book in my Kavita library still says epub at this point with no additional files/versions)
5. Try deleting the epub file, leaving only the azw3 file in the folder - library/author/book.azw3
6. Scan - the entire book is removed from Kavita and so far, I can't get it back.

**Expected behavior**
Kavita to pick up both epub and azw3 file types. Or when epub is removed a scan would still pick up the azw3 file from the folder structure.
Also, deleting the book from Kavita library, providing the file was still in the folder structure, on a new scan I'd expect the library to pick it back up again as the source is still there?

**Desktop (please complete the following information):**
 - OS: unraid docker
 - Browser: chrome
 - Version: 0.6.1.0

**Comment by majora2007** [comment]
Kavita does not support azw3 files, so when you delete the epub, Kavita deletes the series.

**Comment by Shadow229** [comment]
Thanks for the quick response! That makes sense. Can you help me understand why if I hit delete from the Kavita library but the file is still in epub format in the folders it doesn't pick it back up on a scan?

**Comment by majora2007** [comment]
It should pick it back up on scan if the file still exists in the same location, however I can see if you have another series at that level, then Kavita will not do any processing on the file as the underlying folder is showing no changes since the last time we scanned it, as Kavita expects one series per folder at library root. 

That would be my guess.

**Comment by Shadow229** [comment]
hmm, it's got weirder still... I had 6 books I'd converted to azw3 and an empty Kavita library. I've just used Calibre to convert them back into epub files, and Kativa has picked up 4 of them. The other 2 are behaving as before, despite having the epub file in the directory, Kavita just wont see it.

I tried removing the azw3 files entirely, putting me back to my original library/author/book.epub, and then removing the Kavita library and readding it, but even a clean scan from scratch with only 1 series per folder isn't picking the files back up? But according to the [wiki](https://wiki.kavitareader.com/en/guides/managing-your-files/scanner) on scanning multiple series under a folder shouldn't be an issue?

I've just read that kindle now support epub so hopefully if I can get the epub working without the conversion I can still get these on the kindle!
